apps/condo/pages/auth/change-password.tsx

```diff
@@ -43,38 +47,66 @@ const ChangePasswordPage: AuthPage = () => {
     const ErrorToFormFieldMsgMapping = {}
 
     const userId = get(auth, ['user', 'id'])
-
+    const authLayoutContext = useContext(AuthLayoutContext)
     const onFinish = (values: typeof initialValues) => {
-        setIsLoading(true)
+        setIsSaving(true)
         const { token, password } = values
         return runMutation({
             mutation: changePassword,
             variables: { data: { token, password } },
-            onFinally: () => {
-                setIsLoading(false)
-                // if (userId) {
-                //     Router.push('/organizations/')
-                // } else {
-                //     Router.push('/auth/signin/')
-                // }
+            onCompleted: ({ data: { result } }) => {
+                const credentials = {
+                    identity: result.email, secret: form.getFieldValue('password'),
+                }
+                reAuth(credentials)
             },
             intl,
             form,
             ErrorToFormFieldMsgMapping,
         }).catch(error => {
-            setIsLoading(false)
+            setIsSaving(false)
+        })
+    }
+    const [signinByEmailMutation] = useMutation(SIGNIN_MUTATION)
```

Comment:
You can use mutation from AuthContext
```ts
const signInByEmail = ({ email, password }) => {
        return runMutation({
            mutation: signinByEmailMutation,
            variables: { email, password },
            onCompleted: () => {
                refetch()
            },
            intl,
        }).catch(error => {
            console.error(error)
        })
    }
```

Reply:
@dkoviazin it was not working for me as I understood because of different fields: email-identity, password-secret

Reply:
This is a bug probably

Reply:
You can fix signInByEmail to pass identity instead of email - or remove it  from auth context

Reply:
Fixed